src/components/gds-body/_index.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../styles/typography" as *;
+@use "../../styles/colors" as *;
+
+@mixin base {
+  display: block;
```

Comment:
Could this cause issues if we want to use it something that's not a block? Does it need to be set?

Reply:
Yeah, it's probably better to move that to the component's styles.